File: src/main/java/ru/stnk/RestTestAPI/entity/QuotationsData.java
package ru.stnk.RestTestAPI.entity;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public class QuotationsData {
    private QuotationsScreenData screen_data;

    public QuotationsData() {
    }

    public QuotationsScreenData getScreen_data() {
        return screen_data;
    }

    public void setScreen_data(QuotationsScreenData screen_data) {
        this.screen_data = screen_data;
    }

    @Override
    public String toString() {
        return "QuotationsData{" +
                "screen_data=" + screen_data +
                '}';
    }
}

File: src/main/kotlin/ru/stnk/resttestapi/entity/QuotationsData.kt
package ru.stnk.resttestapi.entity

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
class QuotationsData {
    var screen_data: QuotationsScreenData? = null

    override fun toString(): String {
        return "QuotationsData{" +
                "screen_data=" + screen_data +
                '}'.toString()
    }
}
